src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNetIntegration.cs

```diff
@@ -0,0 +1,132 @@
+using System;
+using System.Data;
+using System.Data.Common;
+using System.Threading;
+using System.Threading.Tasks;
+using Datadog.Trace.ExtensionMethods;
+
+namespace Datadog.Trace.ClrProfiler.Integrations
+{
+    /// <summary>
+    /// SqlServer handles tracing System.Data.SqlClient
+    /// </summary>
+    public static class AdoNetIntegration
+    {
+        /// <summary>
+        /// Wrapper method that instruments <see cref="System.Data.Common.DbCommand.ExecuteDbDataReader"/>.
+        /// </summary>
+        /// <param name="this">The <see cref="DbCommand"/> that is references by the "this" pointer in the instrumented method.</param>
+        /// <param name="behavior">A value from <see cref="CommandBehavior"/>.</param>
+        /// <returns>The value returned by the instrumented method.</returns>
+        [InterceptMethod(
+            TargetAssembly = "System.Data", // .NET Framework
+            TargetType = "System.Data.Common.DbCommand",
+            TargetSignature = "20 01")]
```

Comment:
Why do we have to explicitly set the signature for this integration? Coulnd't it be computed automatically by the json generator?

Reply:
This is left over from when we instrumented `ExecuteReader()`. We needed the signature to distinguish between overloads. It looks like `ExecuteDbDataReader()` doesn't have overloads so we can probably drop this signature. I'll test that.

The json generator does figure out the signature for the wrapper method `AdoNetIntegration.ExecuteDbDataReader()`, so we don't need to include that here.

Reply:
The signatures were not needed here, so I removed them.